app/models/message.rb

```diff
@@ -192,6 +192,8 @@ def send_reply
   def reopen_conversation
     return if conversation.muted?
     return unless incoming?
+    # agent bot processor will open the conversation, stay resolved
+    return if conversation.inbox.agent_bot_inbox&.active?
```

Comment:
Shall we mark it as pending instead of returning from here?

Reply:
> Shall we mark it as pending instead of returning from here?

yes doing it now